docs/README.md

```diff
@@ -1,37 +1,43 @@
 # Building Documentation
+## Building locally:
 
-A basic python environment with packages listed in `./requirement.txt` is
-enough to build the docs.
+#### [Build and install cuML](../BUILD.md)
 
-## For building locally:
+#### Use your current cuml dev environment, or build a new one doc building environment.
+```shell script
```

Comment:
Probably the canonical approach to build docs should be `./build.sh pydocs` and we should update teh docs below to reflect that. What do you think about documenting that here, @dantegd ? I believe all of the conda deps should now be in the main dev envs right?

Reply:
Yes, you are right, this readme was severely out of date regarding the pydocs and cppdocs targets of build.sh

Reply:
@drobison00 Could you just remove all mention of the pip based install and rip out requirements.txt here so nobody ever has to fall into the docs build pit you got to enjoy?
We should push people to use `./build.sh pydocs` and conda setup for all docs builds -- anything else seems like a recipe for pain unless the user is an expert in that particular area.

Reply:
Yep, will do.